src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -102,7 +102,7 @@
 	private Plotter myPlotter;
 	private Paper myPaper = new Paper();
 	private Turtle myTurtle = new Turtle();
-	private boolean isMacOS = false;
+	private static boolean isMacOS = false;
```

Comment:
why is this static?

Reply:
because `setSystemLookAndFeel()` is static